### PR TITLE
Update schuelerVZ.net

### PR DESCRIPTION
**Pull request by DavidLiedke**
Was closed in April 2013.

### CODE REVIEW

File: src/chrome/content/rules/SchuelerVZ.xml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<ruleset name="SchuelerVZ.net" default_off="Certificate mismatch">
-<target host="schuelervz.net" />
-  <target host="www.schuelervz.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/SchuelerVZ.xml]: Host is still up and WFM over HTTPS.

**Comment by J0WI** [comment]
@DavidLiedke since this ruleset is `default_off` would you like to fix it?

**Comment by DavidLiedke** [comment]
@J0WI 
I updated the PR.

File: src/chrome/content/rules/SchuelerVZ.xml [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
+<ruleset name="SchuelerVZ.net">
+	<target host="schuelervz.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/SchuelerVZ.xml]: This host times out.
See also `static.pe.schuelervz.net` and `blog.schuelervz.net`.

**Pull request merged**